### ISSUE TITLE
move calibration into mzlib

### ISSUE THREAD

**Issue by lschaffer2** [smith-chem-wisc/mzLib#109]
As calibration is further improved and tested (especially for topdown data), if it was part of mzlib then these improvements could be used in proteoformsuite

**Comment by stefanks** [comment]
Yeah. The inputs would be identified peptide/proteoform spectrum matches, that come from any source. Could come from a MetaMorpheus search, or use a search result from the northwestern software. I think it's workable, I'll think about how to move it from MetaMorpheus in the best way.
